vmf/scripts/mods/vmf/modules/core/network.lua

```diff
@@ -246,32 +262,20 @@ end
 -- ##### Hooks ########################################################################################################
 -- ####################################################################################################################
 
-local function vmf_network_recv(sender, channel_id, rpc_data1, rpc_data2)
-  if not _network_module_is_initialized then
-    return
-  end
-
-  if channel_id == RPC_VMF_REQUEST_CHANNEL_ID then -- rpc_vmf_request
-
-    network_debug("ping", "received", sender)
-
-    send_rpc_vmf_pong(sender)
-
-  elseif channel_id == RPC_VMF_RESPONCE_CHANNEL_ID then -- rpc_vmf_responce
-    -- @TODO: maybe I should protect it from sending by the player who's not in the game?
+local function vmf_received_full_pong(sender, mod_data, rpc_data)
 
     network_debug("pong", "received", sender)
     if _network_debug then
-      vmf:info("[RECEIVED MODS TABLE]: " .. rpc_data1)
-      vmf:info("[RECEIVED RPCS TABLE]: " .. rpc_data2)
+      vmf:info("[RECEIVED MODS TABLE]: " .. mod_data)
+      vmf:info("[RECEIVED RPCS TABLE]: " .. rpc_data)
     end
 
     pcall(function()
 
       local user_rpcs_dictionary = {}
 
-      user_rpcs_dictionary[1] = cjson.decode(rpc_data1) -- mods
-      user_rpcs_dictionary[2] = cjson.decode(rpc_data2) -- rpcs
+      user_rpcs_dictionary[1] = cjson.decode(mod_data) -- mods
+      user_rpcs_dictionary[2] = cjson.decode(rpc_data) -- rpcs
```

Comment:
Nice! I like the descriptive variable names. Maybe the comments can be deleted now since they don't add anything

Reply:
Done